src/ovp_pipeline/ui/view_models.py

```
@@ -5490,9 +5490,53 @@ def build_reader_home_payload(
+    if not candidates:
+        return {}
+    latest = candidates[-1]
+    # Filename is ``YYYY-MM-DD.md`` — extract the date part.
+    date_str = latest.stem
+    # Teaser: skip the heading + blank line, grab the first
+    # paragraph of the digest body.
+    teaser = ""
+    try:
+        body = latest.read_text(encoding="utf-8")
+        for line in body.splitlines():
+            line = line.strip()
```

Ruff (0.15.12)

[warning] `for` loop variable `line` overwritten by assignment target

(PLW2901)